cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -216,8 +216,26 @@
       ]
     },
     {
-      "label": "Worker Nodes",
+      "label": "Auto Scaling for Worker Nodes",
```

Comment:
We should not add "Auto Scaling" here. This section allows setting up autoscaled and fixed cluster configurations